app/generator/generator.go

```diff
@@ -42,6 +42,7 @@ type (
 func NewOrbitGenerator(context *context.OrbitContext) *OrbitGenerator {
 	funcMap := sprig.TxtFuncMap()
 	funcMap["os"] = getOS
+	funcMap["info"] = isInfo
```

Comment:
I feel this would be more conventional to name this `verbose` instead of `info` 🤔 
Running `orbit run --verbose` would feel a bit more explicit than `orbit run --info`